client/app/components/parameters.html

```diff
@@ -1,7 +1,15 @@
-<div class="form-inline bg-white p-5" ng-if="parameters | notEmpty" ui-sortable="{ 'ui-floating': true, 'disabled': !editable }" ng-model="parameters">
-  <div class="form-group" ng-repeat="param in parameters">
+<div class="form-inline bg-white p-5"
+     ng-if="parameters | notEmpty"
+     ui-sortable="{ 'ui-floating': true, 'disabled': !editable }"
+     ng-model="parameters">
+  <div class="form-group m-l-10 m-r-10"
```

Comment:
Yes, at first I tried to add just `m-r-20` only but it looks bad on "mobile" view:
![image](https://user-images.githubusercontent.com/3965316/27025961-ed57c9c2-4f64-11e7-8679-c1353b09d121.png)

Adding 10px on both sides fixes the issue without extra CSS:
![image](https://user-images.githubusercontent.com/3965316/27025840-8b1ab9c2-4f64-11e7-9eb7-15df345f0510.png)

> Btw, did you try putting the labels on top of the input boxes?

Good idea! It looks more cleaner and saves extra space:

![image](https://user-images.githubusercontent.com/3965316/27026675-3d94ab42-4f67-11e7-8f13-115d11a28a8e.png)